### DISCUSSION TITLE
Add Configuration Option to add Tables for snuAddListLinks

### DISCUSSION THREAD

**Discussion by DB-Vadim-Wagner** [arnoudkooi/SN-Utils#497] [Ideas]
Hello,

it would be nice if one could add some custom (log) tables that could be enhanced with `snuAddListLinks` so that you would get links in the message or other fields too.

An Option inside the Settings would be the best way imho.

If wanted I can provide a PR.

**Comment by arnoudkooi** [comment]
I assume these are all OOB tables? Could you share the list you want to add here? I try to avoid adding settings

**Comment by DB-Vadim-Wagner** [comment]
Those tables are custom ones extending `syslog`

**Comment by arnoudkooi** [comment]
I want to see if I can find a generic way, in the meantime you could create a slash command with URL:
`javascript:snuAddListLinks(true);`
If you execute that it will ad-hoc call the function

**Comment by arnoudkooi** [comment]
Did you try this? I have an idea to see if SN Utils can autodetect log tables. trying to keep the settings to a minimal. Moving this to a discussion for now.

**Comment by DB-Vadim-Wagner** [comment]
I tryed it and it works, but it is not that convenient...

**Comment by michaelnow-byteme** [comment]
I think it would be nice if we could use snuAddListLinks on any field. There's plenty of places where table_name_sys_id or the other patterns occur and it would be nice if the jquery was able to select them all.
